### PR TITLE
Add new street address in Berlin.

### PR DESCRIPTION
**Pull request by garloff**
Signed-off-by: Kurt Garloff <kurt@garloff.de>

### CODE REVIEW

File: _i18n/en/footer/imprint.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Imprint 
 
 [Open Source Business Alliance - Bundesverband für digitale Souveränität e.V.](https://osb-alliance.de/)  
-Breitscheidstr. 4, 70174 Stuttgart, Germany
```
Comment (itrich) [inline _i18n/en/footer/imprint.md:4]: By removing the two whitespaces at the end of the line, the line doesn't break.

File: _i18n/en/footer/imprint.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Imprint 
 
 [Open Source Business Alliance - Bundesverband für digitale Souveränität e.V.](https://osb-alliance.de/)  
-Breitscheidstr. 4, 70174 Stuttgart, Germany  
-VR 7217, AG Stuttgart, Chair: Peter H. Ganten
```
Comment (itrich) [inline _i18n/en/footer/imprint.md:5]: See above.

File: _i18n/de/footer/imprint.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Impressum
 
 [Open Source Business Alliance - Bundesverband für digitale Souveränität e.V.](https://osb-alliance.de/)  
-Breitscheidstr. 4, 70174 Stuttgart, Germany
```
Comment (itrich) [inline _i18n/de/footer/imprint.md:4]: By removing the two whitespaces at the end of the line, the line doesn't break.

File: _i18n/de/footer/imprint.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # Impressum
 
 [Open Source Business Alliance - Bundesverband für digitale Souveränität e.V.](https://osb-alliance.de/)  
-Breitscheidstr. 4, 70174 Stuttgart, Germany  
-VR 7217, AG Stuttgart, Chair: Peter H. Ganten
```
Comment (itrich) [inline _i18n/de/footer/imprint.md:5]: See above.

**Review by itrich — CHANGES_REQUESTED** [review]
Thank you very much for this PR! While I changed my own email signature, I totally forgot to change the website. 🙈
 
I highlighted four missing line breaks, which is probably the IDE's fault.

**Comment by garloff** [comment]
I removed the extra whitespace. Also fixed the phone number.

**Comment by itrich** [comment]
Sorry, my review was ambiguous. The whitespaces at the end of the lines need to stay as they add a line break. Some whitespaces were removed in the first commit of this PR. I'll fix this tomorrow!

**Pull request merged**